client/react/component/ClientHistory.js

```diff
@@ -0,0 +1,296 @@
+import React from 'react';
+import { createContainer } from 'meteor/react-meteor-data';
+import { Clients } from '../../../imports/api/clients/clients';
+
+
+const reasons = [
```

Comment:
can you please move `reasons` to a separate file. Otherwise it will be hard to find